src/components/PositionListItem/index.tsx

```diff
@@ -78,6 +78,8 @@ const DoubleArrow = styled.span`
 `
 
 const RangeText = styled.span`
+  display: block;
```

Comment:
you should probably use <div> as the container element if you want block behavior, since spans are by default meant to not impact positioning 🤔

if you use div, you get the display: block for free!

Reply:
sounds like a good idea to me! done